### ISSUE TITLE
Ota Update

### ISSUE THREAD

**Issue by lenisko** [hanatsumi/rakuyomi#63]
Might sounds silly and overcomplicated, but same as KOReader it might be work adding OTA Update to plugin?

Basically downloading a zip if remote VERSION is newer than local, and replacing all Rakuyomi files with new ones & enforcing KOReader restart.

**Comment by hanatsumi** [comment]
Yeah, should be pretty doable. My initial idea would be to prompt the user of a new version in a dialog (or show it somewhere in the interface), and then doing those steps.

Initially, I would lock OTA updates only for minor/patch updates, as an eventual new major version might require manual intervention from the user.
